packages/math-input/src/components/echo-manager.js

```diff
@@ -2,19 +2,23 @@
  * A component that renders and animates the selection state effect effect.
  */
 
-const React = require("react");
-const PropTypes = require("prop-types");
-const {TransitionGroup, CSSTransition} = require("react-transition-group");
-const KeypadButton = require("./keypad-button.js");
+import PropTypes from "prop-types";
+import React from "react";
+import {TransitionGroup, CSSTransition} from "react-transition-group";
+
+import {KeyTypes, EchoAnimationTypes} from "../consts.js";
+
+import * as zIndexes from "./z-indexes.js";
```

Comment:
We should remove z-indexes at some point.  It's such a pain to keep track of them.  In wonder-blocks we use paint order and isolated stacking contexts to avoid the need for global z-indexes.  We should try to do the same with Perseus.

Reply:
I don't disagree... but I have no idea what this means 😂  I'll do some reading. 
> paint order and isolated stacking contexts

Reply:
I tried searching in GDrive, Confluence, and the wonder-blocks and we haven't really documented this anywhere. 😞 

I'll try to write up an explainer when I have some time.